docs/organizations-and-projects.mdx

```diff
@@ -1,7 +1,8 @@
 # Organizations and Projects
 
 ## Organizations
-Organizations are the top-level umbrella for all the projects in a company. A user can create multiple organizations to separate projects under different companies or teams.
+Organizations are collections of projects and where permissions for users are granted. A user may create multiple organizations to separate projects under different companies or teams. *We recommend to create an organization per company*.
```

Comment:
docusaurus has this cool "Tips" feature, you can see it being used here: https://github.com/parca-dev/docs/blob/main/docs/kubernetes.mdx#kubernetes

I think making this a "Tip" makes sense here.

Reply:
TIL!

Here is how it looks now:
<img width="1160" alt="Screenshot 2023-01-03 at 5 20 27 PM" src="https://user-images.githubusercontent.com/1283424/210351914-22b5ccf5-fdd9-480d-949a-8a8d5d71d757.png">